src/components/Error/GenericError.js

```diff
@@ -1,13 +1,167 @@
-import React from 'react'
-import ErrorCard from './ErrorCard'
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  ButtonText,
+  Card,
+  GU,
+  IconDown,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import notFoundImage from '../../assets/dao-not-found.png'
 
 const SUPPORT_URL = 'https://github.com/aragon/aragon/issues/new'
 
-const GenericError = props => (
-  <ErrorCard title="Oops." supportUrl={SUPPORT_URL} {...props}>
-    Something went wrong and the application crashed. Reloading might solve the
-    problem, or you can report the error so we can help.
-  </ErrorCard>
-)
+const GenericError = React.memo(function GenericError({
+  detailsTitle,
+  detailsContent,
+  reportCallback,
+}) {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const toggle = useCallback(() => {
+    setOpened(!opened)
+  }, [opened, setOpened])
+
+  return (
+    <Card
+      css={`
+        display: grid;
+        grid-template-rows: 1fr auto auto auto;
+        grid-template-columns: 1fr;
+        padding: ${5 * GU}px ${6 * GU}px;
+        width: ${72 * GU}px;
+        height: auto;
+        box-shadow: 0px 2px 4px rgba(180, 188, 202, 0.5);
+      `}
+    >
+      <img
+        src={notFoundImage}
+        alt="DAO not found"
+        css={`
+          width: 147px;
+          height: 144px;
+          margin: ${5 * GU}px auto ${1.5 * GU}px;
+        `}
+      />
+      <h1
+        css={`
+          color: ${theme.feedbackSurfaceContent};
+          ${textStyle('title2')};
+          margin-bottom: ${1.5 * GU}px;
+          text-align: center;
+        `}
+      >
+        An unexpected error has occurred
+      </h1>
+      <div
+        css={`
+          color: ${theme.feedbackSurfaceContentSecondary};
+          ${textStyle('body2')};
+          margin: auto;
+          margin-bottom: ${5 * GU}px;
+          max-width: ${52 * GU}px;
+          text-align: center;
+        `}
+      >
+        Something went wrong! Hit reload to restart the app, or you can
+        <ButtonText href={SUPPORT_URL} css="margin: 0 -5px;">
```

Comment:
We might want to use [`horizontalPadding`](https://github.com/aragon/aragon-ui/blob/newstyle/src/components/Button/ButtonText.js#L30) rather than a custom margin here.